apps/meteor/tests/e2e/omnichannel-chat-history.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { faker } from '@faker-js/faker';
+import type { Browser, Page } from '@playwright/test';
+
+import { test, expect } from './utils/test';
+import { OmnichannelLiveChat, HomeOmnichannel } from './page-objects';
+
+const createAuxContext = async (browser: Browser, storageState: string): Promise<{ page: Page; poHomeOmnichannel: HomeOmnichannel }> => {
+	const page = await browser.newPage({ storageState });
+	const poHomeOmnichannel = new HomeOmnichannel(page);
+	await page.goto('/');
+	await page.locator('.main-content').waitFor();
+
+	return { page, poHomeOmnichannel };
+};
```

Comment:
I've seen this snippet of code in a few tests. Shouldn't we move this fn to its own file as a utility and reuse it throughout the tests?

Reply:
Sure! But we could do it in a different PR, maybe we could gather functions we use a lot and create functions for it